diff --git a/lib/components/VBtn/_mixins.scss b/lib/components/VBtn/_mixins.scss
index 9ada6af65a75449d97de9d52e6a6ca499b8224e5..682a8d1f68b049e84e272c845037ab88e5e1c685 100644
--- a/lib/components/VBtn/_mixins.scss
+++ b/lib/components/VBtn/_mixins.scss
@@ -10,7 +10,7 @@ @mixin button-sizes ($map: $button-sizes, $immediate: false) {
     $size: map.get($map, 'font-size') + math.div(2 * $multiplier, 16);
     $height: map.get($map, 'height') + (settings.$size-scale * $multiplier);
 
-    #{if($immediate, &, '')}.v-btn--size-#{$sizeName} {
+    #{if(sass($immediate): &; else: '')}.v-btn--size-#{$sizeName} {
       --v-btn-size: #{$size};
       --v-btn-height: #{$height};
       font-size: var(--v-btn-size);
diff --git a/lib/components/VGrid/_mixins.sass b/lib/components/VGrid/_mixins.sass
index f5a8825c539c380ba1de5a9280ec1de939ab48ed..895077e1a887b22b8a12fbdcc4df52c38ed5e2ff 100644
--- a/lib/components/VGrid/_mixins.sass
+++ b/lib/components/VGrid/_mixins.sass
@@ -37,7 +37,7 @@
 
 @mixin make-col-offset($size, $columns: settings.$grid-columns)
   $num: math.div($size, $columns)
-  margin-inline-start: if($num == 0, 0, math.percentage($num))
+  margin-inline-start: if(sass($num == 0): 0; else: math.percentage($num))
 
 @mixin make-grid-columns($columns: settings.$grid-columns, $gutter: settings.$grid-gutter, $breakpoints: settings.$grid-breakpoints)
   // Common properties for all breakpoints
diff --git a/lib/components/VTimeline/_mixins.sass b/lib/components/VTimeline/_mixins.sass
index 49979acbda5c2e8cb6d31ef1ac945ca45cbcb743..40c9e1bc63ee872a7a37787179341dba84afbb92 100644
--- a/lib/components/VTimeline/_mixins.sass
+++ b/lib/components/VTimeline/_mixins.sass
@@ -1,9 +1,9 @@
 @mixin vertical($immediate: false)
-  @at-root .v-timeline--vertical#{if($immediate, &, ' &')}
+  @at-root .v-timeline--vertical#{if(sass($immediate): &; else: ' &')}
     @content
 
 @mixin horizontal($immediate: false)
-  @at-root .v-timeline--horizontal#{if($immediate, &, ' &')}
+  @at-root .v-timeline--horizontal#{if(sass($immediate): &; else: ' &')}
     @content
 
 @mixin timeline-first-item()
diff --git a/lib/styles/tools/_border.sass b/lib/styles/tools/_border.sass
index 045c74eac39bb9b02e4e627165109f7b972a1128..2439813633a40ac8970f8d8c35f2903753c2662b 100644
--- a/lib/styles/tools/_border.sass
+++ b/lib/styles/tools/_border.sass
@@ -1,8 +1,8 @@
 @mixin border($color: null, $style: null, $width: null, $thin-width: false, $important: false)
   &
-    border-color: $color if($important, !important, null)
-    border-style: $style if($important, !important, null)
-    border-width: $width if($important, !important, null)
+    border-color: $color if(sass($important): !important; else: null)
+    border-style: $style if(sass($important): !important; else: null)
+    border-width: $width if(sass($important): !important; else: null)
 
   @if $thin-width
     &--border
diff --git a/lib/styles/tools/_elevation.sass b/lib/styles/tools/_elevation.sass
index 61e7e192449b3ee939467c07399ff28d4971ad2d..cb1469e687b35943afb5d497f4dc005af80845f5 100644
--- a/lib/styles/tools/_elevation.sass
+++ b/lib/styles/tools/_elevation.sass
@@ -3,7 +3,7 @@
 
 @mixin elevation($z, $important: false)
   &
-    box-shadow: map.get(settings.$shadow-key-umbra, $z), map.get(settings.$shadow-key-penumbra, $z), map.get(settings.$shadow-key-ambient, $z) if($important, !important, null)
+    box-shadow: map.get(settings.$shadow-key-umbra, $z), map.get(settings.$shadow-key-penumbra, $z), map.get(settings.$shadow-key-ambient, $z) if(sass($important): !important; else: null)
 
 @mixin elevationTransition($duration: 280ms, $easing: cubic-bezier(0.4, 0, 0.2, 1))
   &
diff --git a/lib/styles/tools/_functions.sass b/lib/styles/tools/_functions.sass
index 17e47df00107d8c8b90fd293a0bbecfce6f806ad..ef9440d1d5fb8cc6709629b299988315f15b5fac 100644
--- a/lib/styles/tools/_functions.sass
+++ b/lib/styles/tools/_functions.sass
@@ -35,7 +35,7 @@
   @for $i from list.length($maps) through 1
     $current-map: list.nth($maps, $i)
     $current-key: list.nth($keys, $i)
-    $current-val: if($i == list.length($maps), $value, $result)
+    $current-val: if(sass($i == list.length($maps)): $value; else: $result)
     $result: map.merge($current-map, ($current-key: $current-val))
 
   // Return result
@@ -49,10 +49,10 @@
 
 @function breakpoint-min($name, $breakpoints)
   $min: map.get($breakpoints, $name)
-  @return if($min != 0, $min, null)
+  @return if(sass($min != 0): $min; else: null)
 
 @function breakpoint-infix($name, $breakpoints)
-  @return if(breakpoint-min($name, $breakpoints) == null, "", "-#{$name}")
+  @return if(sass(breakpoint-min($name, $breakpoints) == null): ""; else: "-#{$name}")
 
 // Adapted from https://gist.github.com/pentzzsolt/4949bbd7691d43d00616dc4f1451cae9#file-non-destructive-map-merge-4-scss
 @function map-deep-merge($parent-map, $child-map)
diff --git a/lib/styles/tools/_position.sass b/lib/styles/tools/_position.sass
index 1e961365c1b02efdc1113a369523b5cd4cc5502a..4de3098b9efae6fab4abf9b05626cf66713227e5 100644
--- a/lib/styles/tools/_position.sass
+++ b/lib/styles/tools/_position.sass
@@ -1,4 +1,4 @@
 @mixin position($positions, $important: false)
   @each $position in $positions
     &--#{$position}
-      position: #{$position} if($important, !important, null)
+      position: #{$position} if(sass($important): !important; else: null)
diff --git a/lib/styles/tools/_radius.sass b/lib/styles/tools/_radius.sass
index 2f0bba08b1ddb227a93119196c12e13eb544b27c..ec7f5ba5326642e037ed5483f18c4f14e45f8599 100644
--- a/lib/styles/tools/_radius.sass
+++ b/lib/styles/tools/_radius.sass
@@ -5,6 +5,6 @@
   &
     // Key exists within the $rounded variable
     @if (map.has-key(settings.$rounded, $r))
-      border-radius: map.get(settings.$rounded, $r) if($important, !important, null)
+      border-radius: map.get(settings.$rounded, $r) if(sass($important): !important; else: null)
     @else
-      border-radius: $r if($important, !important, null)
+      border-radius: $r if(sass($important): !important; else: null)
diff --git a/lib/styles/tools/_rounded.sass b/lib/styles/tools/_rounded.sass
index 84256a0cf11446cc37633374b958b8ee508bbf68..65a8321ab3b6feb866ca4e1413f3fe6dab1e6feb 100644
--- a/lib/styles/tools/_rounded.sass
+++ b/lib/styles/tools/_rounded.sass
@@ -1,3 +1,3 @@
 @mixin rounded($radius: null, $important: false)
   &
-    border-radius: $radius if($important, !important, null)
+    border-radius: $radius if(sass($important): !important; else: null)
diff --git a/lib/styles/tools/_utilities.sass b/lib/styles/tools/_utilities.sass
index d25bbc4b025d5631cbae694f5615020ff963675f..e8a91bece96b6c6860fb48a5798d15aade16d9af 100644
--- a/lib/styles/tools/_utilities.sass
+++ b/lib/styles/tools/_utilities.sass
@@ -43,10 +43,10 @@
 =generate-utility-body($utility, $properties, $value, $infix)
   // Use custom class if present
   $property-class: map.get($utility, class)
-  $property-class: if($property-class, $property-class, list.nth($properties, 1))
+  $property-class: if(sass($property-class): $property-class; else: list.nth($properties, 1))
 
   // Don't prefix if value key is null (eg. with shadow class)
-  $property-class-modifier: if(list.nth($value, 1), "-" + list.nth($value, 1), "")
+  $property-class-modifier: if(sass(list.nth($value, 1)): "-" + list.nth($value, 1); else: "")
 
   $value: list.nth($value, 2)
 
@@ -61,4 +61,4 @@
         // This allows you to conditional skip
         // defining a property as important.
         $unimportant: map.get($utility, unimportant)
-        #{$property}: #{meta.inspect($val) if(list.index($unimportant, $property), null, !important)}
+        #{$property}: #{meta.inspect($val) if(sass(list.index($unimportant, $property)): null; else: !important)}
